src/fetch-installation-token.ts

```diff
@@ -12,7 +13,13 @@ export const fetchInstallationToken = async ({
   privateKey: string;
   repo: string;
 }>): Promise<string> => {
-  const app = createAppAuth({ appId, privateKey });
+  const app = createAppAuth({
+    appId,
+    privateKey,
+    request: request.defaults({
+      baseUrl: process.env["GITHUB_API_URL"] || "http://api.github.com"
```

Comment:
I think the fallback is unnecessary:

```suggestion
      // GITHUB_API_URL is part of GitHub Actions' built-in environment variables.
      // See https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables.
      baseUrl: process.env["GITHUB_API_URL"]
```

Reply:
The official toolkit uses the fallback, but I'm not sure if there is a good reason.
https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts#L49

Perhaps it supports self-hosted runners? 🤷‍♂️

Reply:
We are using self-hosted runners and that variable is present.  I believe it's added in the runner software so it's probably defensive programming.